packages/compass-assistant/src/assistant-chat.spec.tsx

```diff
@@ -11,16 +11,6 @@ import { createMockChat } from '../test/utils';
 import type { AssistantMessage } from './compass-assistant-provider';
 
 describe('AssistantChat', function () {
-  let originalScrollTo: typeof Element.prototype.scrollTo;
```

Comment:
driveby?

Reply:
yeah this was meant to be removed earlier just ended up staying